.github/workflows/publish.yml

```
@@ -9,12 +9,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3.1.0
```

```suggestion
        uses: actions/checkout@v3
```

Can we pin to the major version? thanks!

---

Many thanks for the great advice!